ui/src/app/kvm/views/KVMList/KVMListHeader/KVMActionFormWrapper/KVMActionFormWrapper.tsx

```diff
@@ -13,6 +15,15 @@ const KVMActionFormWrapper = ({
   selectedAction,
   setSelectedAction,
 }: Props): JSX.Element => {
+  const selectedPodIDs = useSelector(podSelectors.selected);
+
+  // If all KVMs are deselected, close the action form.
+  useEffect(() => {
```

Comment:
Is it possible to test this? If so would probably be good to add one.

Reply:
I've added a super basic test that just checks that if no pods are selected the setSelectedAction is called with `null`. I tried writing a more integration-y test (select some pods, open action form, unselect pods => form closes) but I couldn't get it to work D: any ideas?

Reply:
I think the basic test is fine.